src/Pimple/Tests/PimpleTest.php

```diff
@@ -30,6 +30,9 @@
 
 /**
  * @author  Igor Wiedler <igor@wiedler.ch>
+ * group legacy is here just to avoid deprecation notices when running the tests with the C extension
+ * the annotation should be removed in 4.0
+ * @group legacy
```

Comment:
this would forbid getting errors for usage of other deprecated APIs.

Instead, we should put Travis jobs in weak mode for jobs using the extension.